homeassistant/components/fan/xiaomi_miio.py

```diff
@@ -26,7 +26,7 @@
 DATA_KEY = 'fan.xiaomi_miio'
 
 CONF_MODEL = 'model'
-MODEL_AIRPURIFIER_PRO = 'zhimi.airpurifier.v6'
+MODEL_AIRPURIFIER_PRO = 'zhimi.airpurifier.v7'
```

Comment:
I'm not that familiar with this code, but should the old v6 constant also be kept?

Reply:
Yes. This will will brake the V6 support!